mopidy_iris/static/app.js

```diff
@@ -188341,8 +188341,16 @@ var Browse = /*#__PURE__*/function (_React$Component) {
                 subdirectory.icons = ['/iris/assets/backgrounds/browse-itunes.jpg'];
                 break;
 
-              case 'Soma FM':
+              case 'SomaFM':
                 subdirectory.icons = ['/iris/assets/backgrounds/browse-somafm.jpg'];
+                break;
+				
+			  case 'Jellyfin':
+                subdirectory.icons = ['/iris/assets/backgrounds/browse-jellyfin.jpg'];
+                break;
+				
+			  case 'Podcasts':
+                subdirectory.icons = ['/iris/assets/backgrounds/browse-podcasts.jpg'];
```

Comment:
These edits are in the compiled application code and will be overwritten next time it is compiled.

Instead, you want to editing the source file: https://github.com/jaedb/Iris/blob/master/src/js/views/Library/Browse.js#L86-L88

Reply:
Thank you for letting me know where to do the change, I hope now everything is right.
And thank you for your work too, is very useful for me :)